<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWindowCentralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="6,4">
    <item>
     <widget class="QWidget" name="oasisProWidget" native="true">
      <widget class="QPushButton" name="powerButton">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>300</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Power</string>
       </property>
      </widget>
      <widget class="QPushButton" name="downIntButton">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>300</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
      <widget class="QPushButton" name="upIntButton">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>270</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
      <widget class="QLabel" name="intLabel">
       <property name="geometry">
        <rect>
         <x>483</x>
         <y>250</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(196, 160, 0);</string>
       </property>
       <property name="text">
        <string>INT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="checkButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>âœ“</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="bottom3Graphs">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>280</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="canadaLabel">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>300</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(114, 159, 207);</string>
       </property>
       <property name="text">
        <string>canada</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="ProLabel">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>120</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(114, 159, 207);</string>
       </property>
       <property name="text">
        <string>Oasis</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="oasisLabel">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>140</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(114, 159, 207);</string>
       </property>
       <property name="text">
        <string>Pro</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="mindAliveLabel">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>10</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(114, 159, 207);</string>
       </property>
       <property name="text">
        <string>mind alive</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="graphLabel1">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>310</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>|1|</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="powrButtonLight">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>270</y>
         <width>21</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="graphLabel2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>295</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>|2|</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="graphLabel3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>280</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>|3|</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="graphLabel4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>260</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>|4|</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="graphLabel5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>245</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>|5|</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="middle3Graphs">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>230</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="graphLabel6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>230</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>|6|</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="graphLabel7">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>205</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>|7|</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="graphLabel8">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>190</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>|8|</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="top2Graphs">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>190</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>781</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(11, 11, 11);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>111</width>
         <height>25</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>20 Minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>45 Minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>User Designated</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="timSelectionLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);</string>
       </property>
       <property name="text">
        <string>Time Selection</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <zorder>pushButton</zorder>
      <zorder>top2Graphs</zorder>
      <zorder>middle3Graphs</zorder>
      <zorder>powerButton</zorder>
      <zorder>downIntButton</zorder>
      <zorder>upIntButton</zorder>
      <zorder>intLabel</zorder>
      <zorder>checkButton</zorder>
      <zorder>bottom3Graphs</zorder>
      <zorder>canadaLabel</zorder>
      <zorder>ProLabel</zorder>
      <zorder>oasisLabel</zorder>
      <zorder>mindAliveLabel</zorder>
      <zorder>graphLabel1</zorder>
      <zorder>powrButtonLight</zorder>
      <zorder>graphLabel2</zorder>
      <zorder>graphLabel3</zorder>
      <zorder>graphLabel4</zorder>
      <zorder>graphLabel5</zorder>
      <zorder>graphLabel6</zorder>
      <zorder>graphLabel7</zorder>
      <zorder>graphLabel8</zorder>
      <zorder>comboBox</zorder>
      <zorder>timSelectionLabel</zorder>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="adminTab">
       <attribute name="title">
        <string>Admin</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="replayTab">
       <attribute name="title">
        <string>Replays</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
